src/styles/components/_button.scss

```diff
@@ -20,16 +20,31 @@ button {
   &.secondary-outline {
     @include secondary-outline;
   }
+  &.success {
+    @include success;
+  }
+  &.success-outline {
+    @include success-outline;
+  }
   &.disabled {
     @include disabled;
   }
   &.fluid{
     @include fluid;
   }
-  &:focus{
-    outline: 0;
+  &.round {
+    border-radius: 22px;
+  }
+
+  //Icon
+  i{
+    padding-left: 0;
+  }
+  i:only-child{
+    padding-right: 0;
   }
 
+
   // Button Sizes
   &.large {
     @include size($padding-y-large, $padding-x-large, $font-size-large, $line-height-large, $border-radius);
```

Comment:
focus state gives blue outline. We can remove it 
```
button:focus { outline: 0; }
```

Reply:
outline: 0 Accessibility e karsi oldugu icin biraktim

http://www.outlinenone.com/